mvc/index.html

```diff
@@ -5,7 +5,17 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta name="description" content="Overview of the Laminas MVC packages.">
-    <link rel="shortcut icon" href="/img/trademark-laminas-foundation-rgb.svg">
+
+    <link rel="apple-touch-icon" sizes="180x180" href="https://getlaminas.org/images/favicon/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="https://getlaminas.org/images/favicon/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="https://getlaminas.org/images/favicon/favicon-16x16.png">
+    <link rel="manifest" href="https://getlaminas.org/images/favicon/site.webmanifest">
+    <link rel="mask-icon" href="https://getlaminas.org/images/favicon/safari-pinned-tab.svg" color="#5bbad5">
+    <link rel="shortcut icon" href="https://getlaminas.org/images/favicon/favicon.ico">
+    <meta name="msapplication-TileColor" content="#da532c">
+    <meta name="msapplication-config" content="https://getlaminas.org/images/favicon/browserconfig.xml">
+    <meta name="theme-color" content="#ffffff">
```

Comment:
The sub-project "MVC" has a different logo, so the current favicon is wrong here. See also:

- https://github.com/laminas/laminas.github.io/pull/176#issuecomment-1478986847
- https://github.com/laminas/getlaminas.org/pull/136#issuecomment-1488540683

Reply:
Done